modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -35,60 +35,59 @@ This {release} version of the software and programming language should not be us
 
 The {release} release includes the following new features and capabilities:
 
-- {proglang} and the JavaScript library now support floating point numbers (Float).
-- When using the Candid UI, you can view return values in text, UI, and JSON formats.
-+
-You can switch between the formats for representing values by clicking the results bar.
-- Internal updates that are not yet user-visible to support future features.
+=== SDK
 
-== Breaking change
-The {release} release includes the following changes that might require updates to existing programs:
-
-- The name of the built-in {proglang} library has changed from `stdlib` to `base`.
-- All of the module names in the `base` library start with an upper case letter.
-- The `principalId` module has been changed to `Principal`.
+- You can now look up a canister identifier using the command `dfx canister id <canister_name>`
+- The `--check` flag can be used with the `dfx build` command to check whether a canister will build before creating or building the canister
+- Both canister name and identifiers are printed upon canister creation
+- Canister manifest now stores multiple canister IDs associated with local and remote networks as well as network mapping
+- Local network defaults to `127.0.0.1:8000`
 
-=== Modifying existing programs
+=== Tungsten (Developer Network)
 
-If you have programs that rely on modules in the `stdlib` library, you must change the `import` statement to use the new `base` library name.
-For example, you might have previously imported the Array module using the following statement:
+- HTTP authorization and credentials management for onboarded Tungsten users
+- `Tungsten` added as a provider in `dfx.json
+- You can use the `dfx ping` command to access and toggle between local and Tungsten networks
```

Comment:
`dfx ping` doesn't toggle between local and Tungsten. I think we should just use the help string here:
`Ping an Internet Computer and returns its status`

Reply:
Fixed. @p-shahi should we note this as a known issue? That you can't currently toggle?

Reply:
don't think it's actually a feature to toggle with `dfx ping` so we don't need to note it @alexabsmith